### ISSUE TITLE
.

### ISSUE THREAD

**Issue by BESTczm** [github/docs#4628]
## Pre-requisites

- [ ] Prior to submitting documentation, please apply to join the GitHub Technology Partner Program: [partner.github.com/apply](https://partner.github.com/apply?partnershipType=Technology+Partner). Please feel free to proceed once your application is approved.

## Tasks

Please be sure to complete each of the following:

**Third-party product documentation:**

- [ ] MUST follow our [general contributing guidelines](CONTRIBUTING.md) for voice and markup format.
- [ ] MUST emphasize how the third-party product works with GitHub.
- [ ] MUST be written in Markdown format, using [one of the templates provided](contributing/github-partners/README.md#templates)
- [ ] MUST include the name and URL of the GitHub technology partner responsible for maintenance of the documentation being contributed. This should be added via the `contributor.name` and `contributor.URL` properties in the template's YAML frontmatter.
- [ ] MUST be proposed via a pull request to this repo following [the GitHub Flow](https://guides.github.com/introduction/flow/).
- [ ] MUST be located in the root of [the `content` folder](content). Your filename MUST match the GitHub technology partner name, and use the `.md` file extension.

**The `Pull Request`:**

- [ ] MUST reference this issue, e.g. via `closes #<this issue number>`
- [ ] MUST pass the automated CI checks
- [ ] MUST include links to supporting material demonstrating the functionality being documented (this can be a link to a public GitHub repo, _or_ a video / screencast walkthrough)

Once all tasks are completed, please mention `@github/docs-content` for next steps.

/cc @github/partner-engineering for :eyes:

**Comment by docubot** [comment]
This issue appears to have been opened accidentally. I'm going to close it now, but feel free to open a new issue or ask any questions in [discussions](https://github.com/github/docs/discussions)!
